### PR TITLE
core(config): use fr config to construct the legacy config

### PR DESCRIPTION
**Pull request by adamraine**
Previously the fr default config was just the legacy default config with a few modifications. This PR just reverses the relationship so the new default will be all in one file. The FR config now lives in `default-config.js` and the legacy config lives in `legacy-default-config.js`.

In a follow-up PR I am planning to remove the `fraggle-rock/` config folder and replace it with a `legacy` one.

### CODE REVIEW

**Comment by brendankenny** [comment]
> In a follow-up PR I am planning to remove the `fraggle-rock/` config folder and replace it with a `legacy` one.

maybe that should happen in one PR? Presumably that would preserve more blame since a lot of things (like UIStrings) would be unchanged

**Comment by adamraine** [comment]
> maybe that should happen in one PR? Presumably that would preserve more blame since a lot of things (like UIStrings) would be unchanged

Hmm yeah, also looks like the id for all the UI strings gets updated so I will try that.

File: lighthouse-core/config/default-config.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 /**
- * @license Copyright 2018 The Lighthouse Authors. All Rights Reserved.
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
```
Comment (adamraine) [inline lighthouse-core/config/default-config.js:2]: Hmm we should probably consider this a file modification now

File: lighthouse-core/config/legacy-default-config.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  ...defaultConfig.categories['performance'],
+  auditRefs: defaultConfig.categories?.['performance'].auditRefs
+    .filter(auditRef => !unsupportedAuditIds.includes(auditRef.id)) || [],
+};
+
+/** @type {Record<string, LH.Config.Category>} */
+const categories = {
+  ...defaultConfig.categories,
+  'performance': performance,
+};
```
Comment (connorjclark) [inline lighthouse-core/config/legacy-default-config.js:31]: This bit is confusing, a comment would help.

`Remove unsupported audits from the performance category` (alt: do a JSON.stringify copy and modify in place)

and a comment at top of file

`Construct legacy navigation config`

**Review by connorjclark — APPROVED** [review]
LGTM



> In a follow-up PR I am planning to remove the fraggle-rock/ config folder and replace it with a legacy one.

Should do this first: #14105

**Pull request merged**